Modules/Misc/CommunityMedals.cs

```diff
@@ -526,6 +545,13 @@ static void PostSetScore(ref LeaderboardScore __instance, ref ScoreData newData,
             int medalEarned = GetMedalIndex(levelData.levelID, newData._scoreValueMilliseconds * 1000);
             AdjustMaterial(__instance._medal);
 
+            if(hideLeaderboard.Value && medalEarned >= (int)MedalEnum.Dev)
```

Comment:
Same `if` thing as above, though also use the `I` function just for consistency as well. Same with the ace below.